src/components/Line/index.js

```diff
@@ -31,7 +31,7 @@ const Line = ({
       .curve(d3.curveStepAfter)
       .x(d => boundedSeries(xScale(xAxisAccessor(d))))
       .y(d => boundedSeries(yScale(yAccessor(d))));
-    if (!drawPoints && y0Accessor && y1Accessor) {
+    if (drawPoints !== false && y0Accessor && y1Accessor) {
```

Comment:
Wait, oops -- I meant `!== true`
```suggestion
    if (drawPoints !== true && y0Accessor && y1Accessor) {
```